pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -17,29 +17,30 @@ limitations under the License.
 package v1alpha1
 
 import (
-	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
```

Comment:
This was just a mistake on our part, but I can include it in helper documentation I'll be writing for the community repo.